fuzz/examples/README.md

```diff
@@ -0,0 +1,31 @@
+# Fuzz: Example
+This is a feature for fud. It is a tool that automates the process
+of comparing either two input files, or two backend tools.
+
+The input file should be convertible to Calyx, and any backends should be defined in fud to simulate/execute a program.
+For the compare file functionality, two input files to be compared and a data template file are mandatory, while an input for backend tool and number of iteration are optional (icarus-verilog is the default backend tool).
+For the compare backend functionality, an input file as reference, a data template, and two backend tools mandatory, but the number of iteration is optional.
+
```

Comment:
This is just duplicating what's in the root directory's README, right? I think we can delete this.